### PR TITLE
fix(cb2-7473): default DBA test station to hq

### PR DESCRIPTION
**Pull request by gjulien-bjss**
## Ticket title

_The test station for DBA tests should default to swansea. The user should still be able to update it._
[CB2-7473](https://dvsa.atlassian.net/browse/CB2-7473)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/store/test-records/effects/test-records.effects.ts [hunk 1]
```diff
@@ -169,11 +170,16 @@ export class TestResultsEffects {
       ofType(contingencyTestTypeSelected),
       mergeMap(action =>
         of(action).pipe(
-          withLatestFrom(this.store.select(testResultInEdit), this.store.select(selectTestType(action.testType)), this.userService.user$),
+          withLatestFrom(
+            this.store.select(testResultInEdit),
+            this.store.select(selectTestType(action.testType)),
+            this.userService.user$,
+            this.store.select(getTestStationFromProperty('testStationType', TestStationType.HQ))
```
Comment (tahayazici) [inline src/app/store/test-records/effects/test-records.effects.ts:177]: Why would you have store, store, users, store... Nothing wrong with it, but doesn't it hurt your eyes? 😂

**Review by tahayazici — DISMISSED** [review]

File: src/app/store/test-records/effects/test-records.effects.ts (hunk 1 above)
Comment (gjulien-bjss) [reply src/app/store/test-records/effects/test-records.effects.ts:177]: Eh... I don't really mind 😆

**Review by kpearmanbjss — DISMISSED** [review]

File: src/app/store/test-records/effects/test-records.effects.ts (hunk 1 above)
Comment (Ruairidh-BJSS) [reply src/app/store/test-records/effects/test-records.effects.ts:177]: This needs to be fixed

**Review by kpearmanbjss — APPROVED** [review]

File: src/app/resolvers/test-stations/test-stations.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
```
Comment (mcadenas-bjss) [inline src/app/resolvers/test-stations/test-stations.ts:1]: Is the file name missing `.resolver.`?

**Review by Ruairidh-BJSS — APPROVED** [review]

**Review by kpearmanbjss — APPROVED** [review]

**Pull request merged**